components/server/src/auth/bearer-authenticator.ts

```diff
@@ -87,9 +86,6 @@ export class BearerAuth {
 
         const { user, scopes } = await this.userAndScopesFromToken(token);
 
-        const resourceGuard = new TokenResourceGuard(user.id, scopes);
-        (req as WithResourceAccessGuard).resourceGuard = resourceGuard;
-
```

Comment:
This was the only use of the TokenResourceGurard, but if I'm not completely blind it was assigned and then down below overwritten again.

Reply:
Ok, I'm blind ist' functionGurard vs resourceGurad 😬